Index: all.compressed.js
===================================================================
--- all.compressed.js	(revision 184120)
+++ all.compressed.js	(working copy)
@@ -1485,12 +1485,15 @@
 });
 for(var key in _38){
 var pt=this.sgfCoordToPoint(key);
+if(pt.x!=null){
 if(l==null||pt.x<l){
 l=pt.x;
 }
 if(r==null||pt.x>r){
 r=pt.x;
 }
+}
+if(pt.y!=null){
 if(t==null||pt.y<t){
 t=pt.y;
 }
@@ -1498,6 +1501,7 @@
 b=pt.y;
 }
 }
+}
 this.cropParams.width=r-l+1;
 this.cropParams.height=b-t+1;
 this.cropParams.left=l;
@@ -1515,12 +1519,12 @@
 this.cropParams.height+=_42;
 this.cropParams.top-=_42;
 }
-for(var _43=pad;r+_43>_34;_43--){
+for(var _43=pad;r+_43>_34-1;_43--){
 }
 if(_43){
 this.cropParams.width+=_43;
 }
-for(var _44=pad;b+_44>_34;_44--){
+for(var _44=pad;b+_44>_34-1;_44--){
 }
 if(_44){
 this.cropParams.height+=_44;
Index: player.js
===================================================================
--- player.js	(revision 184120)
+++ player.js	(working copy)
@@ -610,10 +610,14 @@
         // nab the outermost points
         for (var key in points) {
             var pt = this.sgfCoordToPoint(key);
-            if (l == null || pt.x < l) l = pt.x;
-            if (r == null || pt.x > r) r = pt.x;
-            if (t == null || pt.y < t) t = pt.y;
-            if (b == null || pt.y > b) b = pt.y;
+            if (pt.x != null) {
+                if (l == null || pt.x < l) l = pt.x;
+                if (r == null || pt.x > r) r = pt.x;
+            }
+            if (pt.y != null) {
+                if (t == null || pt.y < t) t = pt.y;
+                if (b == null || pt.y > b) b = pt.y;
+            }
         }
         this.cropParams.width = r - l + 1;
         this.cropParams.height = b - t + 1;
@@ -625,9 +629,9 @@
         if (lpad) { this.cropParams.width += lpad; this.cropParams.left -= lpad; }
         for (var tpad = pad; t - tpad < 0; tpad--) {};
         if (tpad) { this.cropParams.height += tpad; this.cropParams.top -= tpad; }
-        for (var rpad = pad; r + rpad > size; rpad--) {};
+        for (var rpad = pad; r + rpad > size-1; rpad--) {};
         if (rpad) { this.cropParams.width += rpad; }
-        for (var bpad = pad; b + bpad > size; bpad--) {};
+        for (var bpad = pad; b + bpad > size-1; bpad--) {};
         if (bpad) { this.cropParams.height += bpad; }
     },
 
