diff -r -U 3 /home/tom/tarballs/eidogo-player-1.2/player/js/all.compressed.js eidogo-player-1.2/player/js/all.compressed.js
--- /home/tom/tarballs/eidogo-player-1.2/player/js/all.compressed.js	2008-04-12 03:13:49.000000000 -0400
+++ eidogo-player-1.2/player/js/all.compressed.js	2009-09-16 19:00:07.000000000 -0400
@@ -2706,7 +2706,7 @@
 location.href=this.downloadUrl+this.gameName;
 }else{
 if(_c){
-location.href="data:text/plain,"+encodeURIComponent(this.cursor.getGameRoot().toSgf());
+location.href="data:application/x-go-sgf,"+encodeURIComponent(this.cursor.getGameRoot().toSgf());
 }
 }
 },save:function(evt){
diff -r -U 3 /home/tom/tarballs/eidogo-player-1.2/player/js/player.js eidogo-player-1.2/player/js/player.js
--- /home/tom/tarballs/eidogo-player-1.2/player/js/player.js	2008-04-12 03:13:49.000000000 -0400
+++ eidogo-player-1.2/player/js/player.js	2009-09-16 18:57:26.000000000 -0400
@@ -2055,7 +2055,7 @@
             }
             location.href = this.downloadUrl + this.gameName;
         } else if (isMoz) {
-            location.href = "data:text/plain," +
+            location.href = "data:application/x-go-sgf," +
                 encodeURIComponent(this.cursor.getGameRoot().toSgf());
         }
     },
